experimental/framework/tree-react-api/src/simpleIdentifier.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+let counter = 0;
+
+const idMap = new WeakMap<object, number>();
+
+/**
+ * Associates a unique number with an object.
+ * @remarks
+ * The id number is tied to the object identity, not the objects contents: modifying the object will not cause it to get a different id.
+ *
+ * This can be handy for generating [keys for React lists](https://react.dev/learn/rendering-lists#where-to-get-your-key) from TreeNodes.
```

Comment:
```suggestion
 * This can be handy for generating {@link https://react.dev/learn/rendering-lists#where-to-get-your-key | keys for React lists} from TreeNodes.
```